modin/_compat/pandas_api/latest/dataframe.py

```diff
@@ -193,6 +329,41 @@ def sum(
             **kwargs,
         )
 
+    def to_gbq(
+        self,
+        destination_table,
+        project_id=None,
+        chunksize=None,
+        reauth=False,
+        if_exists="fail",
+        auth_local_webserver=True,
+        table_schema=None,
+        location=None,
+        progress_bar=True,
+        credentials=None,
+    ):
+        return self._to_gbq(
+            destination_table=destination_table,
+            project_id=project_id,
+            chunksize=chunksize,
+            reauth=reauth,
+            if_exists=if_exists,
+            auth_local_webserver=auth_local_webserver,
+            table_schema=table_schema,
+            location=location,
+            progress_bar=progress_bar,
+            credentials=credentials,
+        )
+
+    def to_orc(self, path=None, *, engine="pyarrow", index=None, engine_kwargs=None):
```

Comment:
We should add test for that. Issue is enough for now.

Reply:
will add

Reply:
https://github.com/modin-project/modin/issues/5085